content/reference-transaction-format.md

```diff
@@ -157,6 +178,16 @@ After a transaction has been submitted, if it gets accepted into a validated led
 }
 ```
 
+
+### Multi-Signing ###
+
+Multi-signing in Ripple is the act of [authorizing transactions](#authorizing-transactions) for the Ripple Consensus Ledger by using a combination of multiple secret keys. Multi-signing is due to be enabled by an [Amendment](concept-amendments.html) to the Ripple Consensus Protocol. You can use multi-signing in addition to, or instead of, a master key, a [regular key](#setregularkey), or both.
+
+The [SignerListSet transaction](#signerlistset) defines which addresses can authorize transactions from your address. You can include up to 8 addresses in a SignerList. You can achieve several different configurations using the weight and quorum values of the SignerList.
```

Comment:
Saying "You can achieve several different configurations" is a bit blurry to me.  Consider something along the lines of "You can give your account one of several different permission models by choosing different the weight and quorum values in the SignerList."  Does "permission models" come close to what the SignerList controls?  Hmmmm...